xbmc/guilib/guiinfo/PlayerGUIInfo.cpp

```diff
@@ -528,6 +528,9 @@ bool CPlayerGUIInfo::GetBool(bool& value, const CGUIListItem *gitem, int context
     case PLAYER_HASDURATION:
       value = g_application.GetTotalTime() > 0;
       return true;
+    case PLAYER_FRAMEADVANCE:
```

Comment:
To get this working you need to extend the guiinfomanager player_labels infomap: https://github.com/xbmc/xbmc/blob/master/xbmc/GUIInfoManager.cpp#L398

And please don't forget to add doxy for the new infobool above the map.

And it would be even more nice (though unrelated) if you could add doxy for the last few bools/labels you've added in recent PRs. It seems that we forgot to add doxy for those new labels/bools- namely for:   

 { "tempoenabled", PLAYER_SUPPORTS_TEMPO},
{ "istempo", PLAYER_IS_TEMPO},
{ "playspeed", PLAYER_PLAYSPEED},
{ "hasprograms", PLAYER_HAS_PROGRAMS},
{ "hasresolutions", PLAYER_HAS_RESOLUTIONS}

Reply:
Please also add the `needs-wiki` label so we can track what needs to be added there

Reply:
> To get this working you need to extend the guiinfomanager player_labels infomap: 

this is part of the PR